dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/executor/plugin/dynamic/DynamicAsyncTaskExecuteFunction.java

```diff
@@ -143,7 +142,7 @@ private void setOutputParameters() {
 
         List<Property> taskPropertyList = new ArrayList<>(JSONUtils.toList(taskInstance.getVarPool(), Property.class));
         taskPropertyList.add(property);
-        logicTask.getTaskParameters().setVarPool(JSONUtils.toJsonString(taskPropertyList));
+        // logicTask.getTaskParameters().setVarPool(JSONUtils.toJsonString(taskPropertyList));
```

Comment:
Why comment out this code?

Reply:
`TaskParameters` doesn't exposed to outside, since Dynamic task will be removed by https://github.com/apache/dolphinscheduler/pull/16842, so this pr doesn't fix this.